Doc/c-api/unicode.rst

```diff
@@ -1396,6 +1396,16 @@ They all return ``NULL`` or ``-1`` if an exception occurs.
    :c:func:`PyErr_Occurred` to check for errors.
 
 
+.. c:function:: int PyUnicode_EqualToString(PyObject *unicode, const char *string)
+
+   Compare a Unicode object with a UTF-8 encoded C string and return true
+   if they are equal and false otherwise.
+
+   This function does not raise exceptions.
```

Comment:
Can you please document the behavior if *string* is not UTF-8 encoded but encoded to Latin1 or something else? What happens? Crash? Undefined behavior?

Reply:
Good, it's a sane behavior.